R/ensure-format.R

```diff
@@ -138,7 +138,7 @@ get_write_function <- function(format, compression, chunk_size = NULL) {
   if (format == "feather") {
     return(function(...) arrow::write_feather(..., chunk_size = chunk_size %||% 65536L, compression = compression))
   } else if (format == "parquet") {
-    return(function(...) arrow::write_parquet(..., chunk_size = chunk_size, compression = compression))
+    return(function(...) arrow::write_parquet(make_table_parquet_compatible(...), chunk_size = chunk_size, compression = compression))
```

Comment:
I suspect that this isn't necessary (since it's just the TPC-H benchmarks that are failing and the changes in `custom-duckdb.R` should cover that, it's not awful to have an additional guard here when making these files.